packages/cli/package.json

```diff
@@ -31,25 +31,25 @@
   "author": "",
   "dependencies": {
     "@clack/prompts": "^0.8.2",
-    "@mastra/core": "^",
+    "@mastra/core": "0.1.27-alpha.20",
```

Comment:
Do we want to pin a specific version here? It seems like we want to figure out a way to use the workspace version but on publish pin to the latest version.

Reply:
i remember Abhi mentioning something related to that.

the workspace version seems to fail the CI test so we have to use the specific version or "^"